contracts/src/factory/facets/architect/Architect.sol

```diff
@@ -40,7 +40,7 @@ contract Architect is
   //                            Space
   // =============================================================
   function createSpace(
-    SpaceInfo memory spaceInfo
+    SpaceInfo calldata spaceInfo
```

Comment:
Nice improvement. It seems spaceInfo is read-only so we can use calldata. Curious why not declare spaceInfo arguments as `calldata` all the way down the stack to _createSpace in [ArchitectBase](https://github.com/river-build/river/blob/main/contracts/src/factory/facets/architect/ArchitectBase.sol#L35) ?

Reply:
The arguments to internal functions in the base contract were also changed to calldata.